benchto-driver/src/main/java/io/trino/benchto/driver/utils/PermutationUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.benchto.driver.utils;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class PermutationUtils
+{
+    public static Integer[] preparePermutation(int size, int permutationId)
+    {
+        List<Integer> intList = IntStream.range(0, size).boxed().collect(Collectors.toList());
```

Comment:
static import `toList` here (simialr in other places that use `toList` or `toImmutableList`)